meetings/2024-10-22.md

```diff
@@ -1,11 +1,67 @@
+* Guiding questions have been shared in github forums.
+* Today, group is to focus on guiding principles for standard changes.
+
+#### Principles for change
+
+* Discussed proposed principles for change based on Github forum discussions.
+* The transparency of the process should also be explicit.
+* Broadening participation and outreach to affected groups is an important principle.
+* Different types of changes - use case driven, technical/maintenance driven, changes mandated by changes to other standards. 
+
+#### Principles for the Standard?
+
+* Discussed proposed principles for the standard based on Github forum discussions.
+* What is the role of the standard in the initiative and where do we want it to go?
+* Noted that there is an opportunity to bring this into the 2025-2030 Strategic Planning process.
+* Different meanings of "modularity" in the standard, and possible impact - complexity vs innovation.
+* Noted that "splitting the standard up" away from the activity model (ie: into data modules) is not currently desirable or practical.
+* Shared examples of extensions from the [Open Contracting standard](https://extensions.open-contracting.org/en/).
+* Comparability - how comparable should IATI data be from different reporting organisations?
+
+#### Responsibilities
+
+* What roles and responsibilities should the various stakeholders have in the [change process](https://iatistandard.org/en/iati-standard/upgrades/how-we-manage-the-standard/)?
+* Example provided from GitHub forum, around how the Major, Minor & Patch mechanisms already imply roles and responsibilities.
+* Emphasis on pragmatism in terms of us all making changes in the most effective way - rather than be held up by over complicated process.
+* Example from [360 Giving standard](https://www.threesixtygiving.org/data-standard/governance-of-the-data-standard/).
+* How should change requests be received?
+* Need to balance the process - checks and balances without burden. Transparency and good communication can mitigate this.
+
+#### Translation
+
+* How do we address accessibility of engaging with the change process, including but not limited to translation?
+* Proposed changes should use short, non technical language as far as possible.
+* Change summary one-pagers should be translated.
+* Keep an eye on pragmaticism, too.
+
+#### Guidance
+
+* The guidance is separate from the standard and not currently change controlled. Should this be considered in our process?
+* Noted that there is [guidance on normative and non-normative guidance](https://iatistandard.org/en/iati-standard/upgrades/guidance-updates/) already in place.
+* Agreed on flexibility to change, and need to create shorter, clearer guidance than gives context.
+* Also need for user guidance - if you use the data, this is what it means and how you should use it. For example, a standard way to split multi sector transactions.
+* Common pitfalls in the data.
```

Comment:
@IsabelBirds @stevieflow I'm not sure what this means; are we saying that the role of guidance is to help people avoid common pitfalls?

Reply:
This was a comment that user guidance is more useful if it includes common pitfalls/things to avoid doing instead of trying to describe everything that can be done with the data.